packages/mwp-app-render/src/components/ApolloProvider.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ApolloClient, { InMemoryCache } from 'apollo-boost';
+import fetch from 'isomorphic-fetch';
+import { ApolloProvider } from 'react-apollo';
+import { isServer } from '../util/isServer';
+
+const client = new ApolloClient({
+	ssrMode: isServer(),
+	name: 'mup-web',
+	cache: isServer()
```

Comment:
I think you need to make a new client on search server invocation to avoid a memory leak